source/Ship.cpp

```diff
@@ -1392,13 +1393,20 @@ vector<string> Ship::FlightCheck() const
 			checks.emplace_back("solar power?");
 		if(fuel < 0.)
 			checks.emplace_back("fuel?");
-		if(!canBeCarried)
+		if(canBeCarried && !intrasolar)
```

Comment:
```suggestion
		if(!canBeCarried && !intrasolar)
```

This code section originally applied these two checks to ships that cannot be carried. Meaning ships designed for HD travel. As such, I'm curious as to the logic of switching this check to ships that can be carried and don't have the intrasolar attribute. 

Possible combinations (all ships will be one of these 4 situations):
canBeCarried + Intrasolar = no fuel or HD checks
!canBeCarried + Intrasolar = no fuel or HD checks
canBeCarried + !Intrasolar = no fuel or HD checks
!canBeCarried + !Intrasolar = Do the fuel and HD checks

As such, the correct situation for this if statement to be true and do the HD and fuel checks would be when the ship has both !canBeCarried and !Intrasolar.